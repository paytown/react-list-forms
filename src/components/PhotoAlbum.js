import React from 'react';
import PropTypes from 'prop-types';
import Photos from './Photos';

function PhotoAlbum({ title, photos }) {
  return (
    <div>
      <h3>{title}</h3>
      <Photos photos={photos} />
    </div>
  );
}

PhotoAlbum.propTypes = {
  title: PropTypes.string,
  photos: PropTypes.array.isRequired
};

PhotoAlbum.defaultProps = {
  title: 'My Cool Dogs'
};

export default PhotoAlbum;
