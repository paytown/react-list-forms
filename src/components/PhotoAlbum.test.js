import React from 'react';
import { shallow } from 'enzyme';
import PhotoAlbum from './PhotoAlbum';

describe('it displays cool dogs', () => {
  it('lets see those dogs', () => {
    const wrapper = shallow(<PhotoAlbum photos={[]}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
