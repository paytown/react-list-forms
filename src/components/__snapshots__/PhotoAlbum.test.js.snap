// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it displays cool dogs lets see those dogs 1`] = `
<div>
  <h3>
    My Cool Dogs
  </h3>
  <Photos
    photos={Array []}
  />
</div>
`;
